research/sentiment_analysis/sentiment_model.py

```diff
@@ -25,9 +31,9 @@ def __init__(self, emb_dim, num_words, sentence_length, hid_dim,
       tf.keras.models.Model: A Keras model.
     """
 
-    layer = tf.keras.layers.Input(shape=(sentence_length,), dtype=tf.int32)
+    sen = tf.keras.layers.Input(shape=(sentence_length,), dtype=tf.int32)
```

Comment:
What does sen stand for here? Can you use a clearer var name?

Reply:
ack